events-processor/main.go

```diff
@@ -25,5 +52,5 @@ func main() {
 	defer sentry.Flush(2 * time.Second)
 
 	// start processing events & loop forever
-	processors.StartProcessingEvents()
+	processors.StartProcessingEvents(ctx, logger, otelEndpoint != "")
```

Comment:
I would use a config here for clarity:

```suggestion
	processors.StartProcessingEvents(ctx, processors.Config{logger: logger, useTelemetry: otelEndpoint != ""})
```

Reply:
It's better indeed! Good catch